redfish/rules/dell_redfish_1.17.0.yml

```diff
@@ -1,449 +1,484 @@
-    Pointer: /StorageControllers/{storagecontroller}/Status/Health
-    Type: health
-  - Name: systems_storage_storagecontrollers_status_state
-    Pointer: /StorageControllers/{storagecontroller}/Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/Storage/{storage}/Controllers/{controller}
-  Properties:
-  - Name: systems_storage_controllers_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_storage_controllers_status_state
-    Pointer: /Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/Storage/{storage}/Drives/{device}
-  Properties:
-  - Name: systems_storage_drives_failurepredicted
-    Pointer: /FailurePredicted
-    Type: bool
-  - Name: systems_storage_drives_predictedmedialifeleftpercent
-    Pointer: /PredictedMediaLifeLeftPercent
-    Type: number
-  - Name: systems_storage_drives_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_storage_drives_status_state
-    Pointer: /Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/Storage/{storage}/StorageControllers/{controller}
-  Properties:
-  - Name: systems_storage_storagecontrollers_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_storage_storagecontrollers_status_state
-    Pointer: /Status/State
-    Type: state
-  - Name: systems_storage_storagecontrollers_storagecontrollers_status_health
-    Pointer: /StorageControllers/{storagecontroller}/Status/Health
-    Type: health
-  - Name: systems_storage_storagecontrollers_storagecontrollers_status_state
-    Pointer: /StorageControllers/{storagecontroller}/Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/Storage/{storage}/Volumes/{volume}
-  Properties:
-  - Name: systems_storage_volumes_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_storage_volumes_status_state
-    Pointer: /Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/PCIeDevices/{device}
-  Properties:
-  - Name: systems_pciedevices_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_pciedevices_status_state
-    Pointer: /Status/State
-    Type: state
-- Path: /redfish/v1/Systems/{system}/PCIeDevices/{device}/PCIeFunctions/{function}
-  Properties:
-  - Name: systems_pciedevices_pciefunctions_status_health
-    Pointer: /Status/Health
-    Type: health
-  - Name: systems_pciedevices_pciefunctions_status_state
-    Pointer: /Status/State
-    Type: state
-Traverse:
-  Excludes:
-  - /JsonSchemas
-  - /Accounts
-  - /Certificates
-  - /Jobs
-  - /Logs
-  - /Registries
-  - /Roles
-  - /Sessions
-  - /Settings
-  - /AccountService
-  - /CertificateService
-  - /EventService
-  - /JobService
-  - /LogServices
-  - /SessionService
-  - /TaskService
-  - /TelemetryService
-  - /UpdateService
-  - /LicenseService
-  - /Power/
-  - /Power#/
-  - /Thermal#/
-  - /Assembly#/
-  - /redfish/v1/Chassis/$
-  Root: /redfish/v1
+Traverse:
```

Comment:
I converted the line feed code from CRLF to LF.
Please `Hide whitespace` when reviewing.